patches/node/inspector_add_network_initiator_in_inspector_protocol.patch

```diff
@@ -0,0 +1,322 @@
+       'inputs': [
+         'node_protocol_config.json',
++        'node_protocol.pdl',
+         '<(SHARED_INTERMEDIATE_DIR)/src/node_protocol.json',
+         '<@(node_protocol_files)',
+         '<(protocol_tool_path)/code_generator.py',
+diff --git a/src/inspector/node_protocol.pdl b/src/inspector/node_protocol.pdl
+index d5b50dc81b40f1d7324d70dabee38eb30956f8d9..2fe6634ad7e27882d62c6c421c15344139ce4e3f 100644
+--- a/src/inspector/node_protocol.pdl
++++ b/src/inspector/node_protocol.pdl
+@@ -101,6 +101,8 @@ experimental domain NodeWorker
+ # Partial support for Network domain of ChromeDevTools Protocol.
+ # https://chromedevtools.github.io/devtools-protocol/tot/Network
+ experimental domain Network
++  depends on Runtime
++
+   # Resource type as it was perceived by the rendering engine.
+   type ResourceType extends string
+     enum
+@@ -132,6 +134,31 @@ experimental domain Network
+   # Monotonically increasing time in seconds since an arbitrary point in the past.
+   type MonotonicTime extends number
+ 
++  # Information about the request initiator.
++  type Initiator extends object
++    properties
++      # Type of this initiator.
++      enum type
++        parser
++        script
++        preload
++        SignedExchange
++        preflight
++        other
++      # Initiator JavaScript stack trace, set for Script only.
++      # Requires the Debugger domain to be enabled.
++      optional Runtime.StackTrace stack
++      # Initiator URL, set for Parser type or for Script type (when script is importing module) or for SignedExchange type.
++      optional string url
++      # Initiator line number, set for Parser type or for Script type (when script is importing
++      # module) (0-based).
++      optional number lineNumber
++      # Initiator column number, set for Parser type or for Script type (when script is importing
++      # module) (0-based).
++      optional number columnNumber
++      # Set if another request triggered this request (e.g. preflight).
++      optional RequestId requestId
++
+   # HTTP request data.
+   type Request extends object
+     properties
+@@ -163,6 +190,8 @@ experimental domain Network
+       RequestId requestId
+       # Request data.
+       Request request
++      # Request initiator.
++      Initiator initiator
+       # Timestamp.
+       MonotonicTime timestamp
+       # Timestamp.
+diff --git a/src/inspector/node_protocol_config.json b/src/inspector/node_protocol_config.json
+index 959a1f4eb1b67a18b62034c5657b6ff9402f222c..6c3cbdd4cb95a70333019dd1e2c5f445c14b3a22 100644
+--- a/src/inspector/node_protocol_config.json
++++ b/src/inspector/node_protocol_config.json
+@@ -5,6 +5,17 @@
+         "output": "node/inspector/protocol",
+         "namespace": ["node", "inspector", "protocol"]
+     },
++    "imported": {
++        "path": "../../deps/v8/include/js_protocol.pdl",
++        "header": "<v8-inspector-protocol.h>",
++        "namespace": ["v8_inspector", "protocol"],
```

Comment:
this new section isn't playing well with `electron_node/tools/inspector_protocol/code_generator.py`